### PR TITLE
[torch_glow] Removal of batch_norm constraint of input rank, to suppo…

### PR DESCRIPTION
**Pull request by dangus36**
Removal of constraint of input rank, to support Sparse input. Even if the input is sparse tensor, BatchNorm can work directly by applying normalization on the values. The checks for shape validation are already added to verify().

### CODE REVIEW

File: torch_glow/src/PyTorchModelLoader.cpp [hunk 1]
```diff
@@ -3535,11 +3535,6 @@ Error PyTorchModelLoader::loadBatchNorm(const torch::jit::Node *ptNode) {
                                            inputs[BatchNormInputs::training])));
   RETURN_ERR_IF_NOT(training == false, "Don't support BatchNorm training yet.");
 
-  RETURN_ERR_IF_NOT(
-      input.dims().size() == numDims + 2,
-      glow::strFormat("Number input dimensions must be equal to %d, got %lu",
-                      numDims + 2, input.dims().size()));
-
   size_t numChannels = input.dims()[1];
```
Comment (zrphercule) [inline torch_glow/src/PyTorchModelLoader.cpp:3538]: Here we are relying on the knowledge of "second dim is channel." If input.dims.size() != numDims + 2, does this still satisfy?

**Comment by dangus36** [comment]
Commenting so it doesn't get stale
